go.mod

```diff
@@ -3,6 +3,7 @@ module sigs.k8s.io/gateway-api
 go 1.24.0
 
 require (
+	github.com/eclipse/paho.mqtt.golang v1.5.1
```

Comment:
side note (no action from you @phuhung273 ) 

@robscott one more reason we should move fast removing conformance from main go.mod :D